src/quorum.c

```diff
@@ -112,20 +112,22 @@ static ktime_t random_to(u32 lo, u32 hi)
 /*
  * The caller is about to read all the quorum blocks.  We invalidate any
  * cached blocks and issue one large contiguous read to repopulate the
- * cache.  The caller then uses normal sb_bread to read each block.  I'm
+ * cache.  The caller then uses normal __bread to read each block.  I'm
  * not a huge fan of the plug but I couldn't get the individual
  * readahead requests merged without it.
  */
 static void readahead_quorum_blocks(struct super_block *sb)
 {
+	struct scoutfs_sb_info *sbi = SCOUTFS_SB(sb);
 	struct buffer_head *bh;
 	struct blk_plug plug;
 	int i;
 
 	blk_start_plug(&plug);
 
 	for (i = 0; i < SCOUTFS_QUORUM_BLOCKS; i++) {
-		bh = sb_getblk(sb, SCOUTFS_QUORUM_BLKNO + i);
+		bh = __getblk(sbi->meta_bdev, SCOUTFS_QUORUM_BLKNO + i,
+			      sbi->meta_bdev->bd_block_size);
```

Comment:
The SCOUTFS_BLOCK_SM_SIZE block size is hard wired into the on-disk format, might as well just use it here instead of the bd_block_size (I wonder if you can change the block size any old time at run time :)).